wrappers/python/pyrs_frame.cpp

```diff
@@ -162,7 +162,14 @@ void init_frame(py::module &m) {
         .def("get_bits_per_pixel", &rs2::video_frame::get_bits_per_pixel, "Retrieve bits per pixel.")
         .def_property_readonly("bits_per_pixel", &rs2::video_frame::get_bits_per_pixel, "Bits per pixel. Identical to calling get_bits_per_pixel.")
         .def("get_bytes_per_pixel", &rs2::video_frame::get_bytes_per_pixel, "Retrieve bytes per pixel.")
-        .def_property_readonly("bytes_per_pixel", &rs2::video_frame::get_bytes_per_pixel, "Bytes per pixel. Identical to calling get_bytes_per_pixel.");
+        .def_property_readonly("bytes_per_pixel", &rs2::video_frame::get_bytes_per_pixel, "Bytes per pixel. Identical to calling get_bytes_per_pixel.")
+        .def("extract_target_dimensions", [](const rs2::video_frame& self, int target_type)->std::array<float, 4>
```

Comment:
- The output array may vary according to the target type. Use std::vector to return the results.
- Pass the target type as enum (requires to define BIND_ENUM(... in c_files.cpp